content-style-guide/01-voice-and-tone.md

```diff
@@ -0,0 +1,98 @@
+to be personal and inviting. To achieve this, consider addressing your reader like
+a friend.
+
+You might say: _"You're welcome to chat with us about Node.js core features on our
+Github issues list!"_
+
+Instead of: _"The GitHub issues list is the place for discussion of Node.js core
+features."_
+
+## Our tone
+
+Node.js’ tone is usually informal. When writing content or a response, being clear
+and direct is always more important than being entertaining.
+
+Actions are also part of tone. A request for help going ignored for many weeks could be perceived as unwanted and unworthy. A quick response, even if an issue itself remains unresolved, shows compassion and thoughtfulness.
+
+It will help to know the audience and the situation when deciding on the right tone. When writing for an audience, think about asking yourself these six questions:
+
+* Who will be reading this content or post? _This is your intended audience._
+* What does the reader want? _This is their goal._
+* How familar is the reader with the content? _This lets you tailor your answer
+to them._
+* What situation is the reader in? _This helps you understand why they are here._
+* What emotion is the reader feeling at this moment? _This helps you empathise,
+to help better._
+* How can you maintain (or better yet, improve) the reader's state of mind? _This
+is your goal._
+
+When you understand the context to which the conversation belongs, you can adjust
+your tone to fit.
+
+### Examples of tone
+
+The [Issues](https://github.com/nodejs/help/issues) tracker of the Node.js Help
+repo  is an example of an area where tone is especially important. Users of Node.js
+are opening issues to solicit help. It's often difficult to ask for help! You can
+make their experience directly interacting with Node.js a good one by putting thought
+into using a tone appropriate to the request.
+
+They might say: _"How do I load another .js file and run that new .js file in a new
+global context?..."_
+
+You might respond: _"If you really want to do this, you will need to put a wrapper
+function around your code, which provides implementations of require, module etc.
+You might want to take a look at..."_
+
+The response above is direct, yet respectful. It does not make assumptions about
+the best action; that was not asked for. It is straightforward and just provides
+the requested answer. It invites, rather than commands, the person to make use of
+a suggested resource.
+
+## Quick style tips ⚡️
+
+These are key elements to get started with writing in Node.js’ voice. For details, read Mailchimp's excellently written [Grammar and Mechanics Styleguide](https://styleguide.mailchimp.com/grammar-and-mechanics/), which inspired this section.
+
+* **Do: Active voice** Use active voice. Avoid passive voice.
+* **Don't: Jargon (and slang)** Write in vanilla English.
+* **Do: Positivity** Use positive language, instead of negative language.
```

Comment:
resources for using positive and negative language?

Reply:
Good point. This one cites sources and has dos vs don'ts. https://www.plainlanguage.gov/guidelines/concise/use-positive-language/

Putting in the next commit.